base/Dockerfile.dataengineering

```diff
@@ -24,20 +24,6 @@ RUN adduser --home /home/${USER} \
 # Copy apt configuration
 COPY config/debian/etc/apt/apt.conf /etc/apt/
 
-COPY                                         \
-  scripts/common/install-rds-certificates.sh \
-  scripts/common/install-sops.sh             \
-  /tmp/scripts/common/
-RUN :                                                         \
-  && apt-get update                                           \
-  # Run common install scripts                                \
-  && apt-get install libopenblas-dev wget git less bash curl openssl coreutils ca-certificates jq tar -y \
-  && chmod +x /tmp/scripts/common/install-rds-certificates.sh \
-  && chmod +x /tmp/scripts/common/install-sops.sh             \
-  && /tmp/scripts/common/install-rds-certificates.sh          \
-  && /tmp/scripts/common/install-sops.sh                      \
-  && ln -s /usr/include/locale.h /usr/include/xlocale.h
```

Comment:
not sure what locale.h and xlocal.h is used for

Reply:
locale.h is a [library](https://www.tutorialspoint.com/c_standard_library/locale_h.htm) that is used to make it easier to set things like date formats, currency, and numerical formats up for a specified region. Be sure that none of your libraries (such as numpy) need it, before you remove it.

Reply:
added the ls back